data_structures/stack/balanced parenthesis using stack in C

```diff
@@ -57,7 +58,7 @@ int isBalanced(char *s) {
         
             
                x=pop();			
-                if( x=='{'&&s[i]!='}')	
+                if( x=='{'&&s[i]!='}')	//to check if the opening bracket of the same type at the closing one is popped.
```

Comment:
Can you put in spaces between the operators and conditions, such as
```c
x == '{' && s[i] != '}'
```
for more readability